gitpod-io/website
# Debugger: JavaScript Langauge Page

## Description

docs: Debug JavaScript Application on Gitpod

## How to test

- Open [`/docs/introduction/languages/javascript`](https://deploy-preview-3204--gitpod-kumquat.netlify.app/docs/introduction/languages/javascript#debug-your-javascript-application-in-gitpod) & See 👀 



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3204"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
